CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [CHANGE] The `chunk.ObjectClient` interface has been renamed `chunk.Client`. A new `chunk.ObjectClient` interface has been created to store raw byte encoded objects in an object backend. The following Object backends (S3/GCS/AzureBlobStore/LocalFS) now are implemented using a generic client in `pkg/chunk/object.ChunkClient` utilizing the new `ObjectClient` interface. #2108
```

Comment:
In the changelog we currently write down only user-facing changes. This shouldn't have any impact on the user, so I would remove this entry at all. What do you think?

Reply:
I removed the entry